src/processing_provider/swmm_extract_results.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+
+"""
+/***************************************************************************
+ QGEP-swmm processing provider
+                              -------------------
+        begin                : 07.2019
+        copyright            : (C) 2019 by ig-group.ch
+        email                : timothee.produit@ig-group.ch
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+#import qgis.utils as qgis_utils
+
+#import datetime
+
+from qgis.core import (
+    #QgsExpression,
+    QgsFeature,
+    #QgsFeatureRequest,
+    QgsFeatureSink,
+    QgsField,
+    QgsFields,
+    #QgsGeometry,
+    #QgsProcessing,
+    #QgsProcessingAlgorithm,
+    QgsProcessingContext,
+    QgsProcessingException,
+    QgsProcessingFeedback,
+    #QgsProcessingParameterString,
+    QgsProcessingParameterFile,
+    #QgsProcessingParameterFileDestination,
+    #QgsWkbTypes,
+    QgsProcessingParameterFeatureSink
+)
+
+from .qgep_algorithm import QgepAlgorithm
+from .QgepSwmm import QgepSwmm
+
+#from ..tools.qgepnetwork import QgepGraphManager
```

Comment:
Can you delete all those unrequired imports?

Reply:
Done